src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -44,29 +43,30 @@ public static OpenTelemetryLoggerOptions AddOtlpExporter(this OpenTelemetryLogge
         Action<OtlpExporterOptions> configure)
         => AddOtlpExporterInternal(loggerOptions, configure);
 
-    private static OpenTelemetryLoggerOptions AddOtlpExporterInternal(
-        OpenTelemetryLoggerOptions loggerOptions,
-        Action<OtlpExporterOptions> configure)
+    /// <summary>
+    /// Adds OTLP Exporter as a configuration to the OpenTelemetry ILoggingBuilder.
```

Comment:
We could probably have a better wording here. Maybe something like:

"Adds OTLP Exporter to the `OpenTelemetryLoggerProvider`."

Reply:
checked with @CodeBlanch offline. Updated to slightly different as OpenTelemetryLoggerProvider is not something end user would be familiar with.